orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CancelStageHandler.kt

```diff
@@ -40,7 +40,7 @@ open class CancelStageHandler
 
   override fun handle(message: CancelStage) {
     message.withStage { stage ->
-      if (stage.getStatus().halt) {
+      if (stage.getStatus().isHalt) {
```

Comment:
I've run into this before with Kotlin and I'm confused why it does this but I don't think there's a way round it without violating expected naming conventions for these kind of methods in Java.